general/networking/nginx.md

```diff
@@ -107,7 +133,7 @@ server {
         # The / at the end is significant.
         # https://www.acunetix.com/blog/articles/a-fresh-look-on-reverse-proxy-related-attacks/
 
-        proxy_pass http://SERVER_IP_ADDRESS:8096/jellyfin/;
+        proxy_pass http://$jellyfin:8096/jellyfin/;
```

Comment:
@dkanada thoughts? we've avoided using environmentals for aliases. I was for it because I'm accustomed to them due to docker, but non-docker and windows people won't necessarily know what they mean.

Reply:
I have no opinion besides picking a standard, since we currently use a wide range of methods for situations like this.